frontend/src/views/toolbox/device/index.vue

```diff
@@ -72,7 +72,7 @@
                             <el-form-item :label="$t('toolbox.device.localTime')" prop="localTime">
                                 <el-input disabled v-model="form.localTime">
                                     <template #append>
-                                        <el-button @click="onChangeLocalTime" icon="Refresh">
+                                        <el-button @click="onChangeLocalTime" icon="Refresh" width="150px">
                                             {{ $t('commons.button.sync') }}
                                         </el-button>
                                     </template>
```

Comment:
There is no irregularity or issue in the provided code snippet. The changes made to increase the button's width from auto to 150px are minor style adjustments that may improve user interface aesthetics without affecting functionality.

Optimization suggestion: If you want to keep the button smaller, maintaining it at its default width (`auto`), ensure that there is enough vertical space around it within the form layout to prevent overlapping with other UI components. You can experiment with different padding settings on the `<el-form>` container if necessary.